### PR TITLE
UIIN-1407: Instance record | Display a new action in Actions menu - Duplicate a MARC bibliographic record

### PR DESCRIPTION
**Pull request by BogdanDenis**
## Purpose
Add ability to duplicate MARC bib records based on existing inventory items.

## Issues
[UIIN-1407](https://issues.folio.org/browse/UIIN-1407)

## Related PRs
https://github.com/folio-org/ui-inventory/pull/1296

## Screenshots
image

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

### CODE REVIEW

**Review by Vladyslav-Velytskyi — APPROVED** [review]

File: src/QuickMarc.test.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+/* eslint-disable react/prop-types */
+
+import React from 'react';
+import { Router } from 'react-router-dom';
+import { render, cleanup } from '@testing-library/react';
```
Comment (Yuliia-Dovhal) [inline src/QuickMarc.test.js:5]:
```suggestion
import { 
  render, 
  cleanup,
} from '@testing-library/react';
```

File: src/QuickMarc.test.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+import QuickMarc from './QuickMarc';
+
+jest.mock('./QuickMarcEditor', () => {
+  return {
+    QuickMarcEditorContainer: ({ action }) => <span>QuickMarcEditorContainer {action}</span>,
+  };
+});
+
+const renderQuickMarc = ({ basePath, onClose, history }) => (render(
```
Comment (Yuliia-Dovhal) [inline src/QuickMarc.test.js:18]:
```suggestion
const renderQuickMarc = ({ 
  basePath, 
  onClose, 
  history,
}) => (render(
```

File: src/QuickMarcEditor/QuickMarcEditor.test.js [hunk 3]
```diff
@@ -1,15 +1,32 @@
+/* eslint-disable react/prop-types */
+
 import React from 'react';
 import { MemoryRouter } from 'react-router-dom';
-import { render, cleanup } from '@testing-library/react';
+import { render, cleanup, fireEvent } from '@testing-library/react';
```
Comment (Yuliia-Dovhal) [inline src/QuickMarcEditor/QuickMarcEditor.test.js:5]:
```suggestion
import { 
  render, 
  cleanup, 
  fireEvent,
} from '@testing-library/react';
```

File: src/QuickMarcEditor/QuickMarcEditor.test.js [hunk 4]
```diff
@@ -86,4 +104,24 @@ describe('Given Quick Marc Editor', () => {
 
     expect(getByText('QuickMarcEditorRows')).toBeDefined();
   });
+
+  describe('When deleting a row', () => {
+    it('Then it should not display ConfirmationModal', () => {
+      const instance = getInstance();
+      const { getByText, queryByText } = renderQuickMarcEditor({
```
Comment (Yuliia-Dovhal) [inline src/QuickMarcEditor/QuickMarcEditor.test.js:111]:
```suggestion
      const { 
        getByText, 
        queryByText,
      } = renderQuickMarcEditor({
```

File: src/QuickMarcEditor/utils.js [hunk 5]
```diff
@@ -24,6 +29,50 @@ export const dehydrateMarcRecordResponse = marcRecordResponse => ({
...
+
+      return false;
+    }
+
+    return true;
+  };
+
+  return {
+    ...marcRecord,
+    records: marcRecord.records.map(field => (shouldRemoveFieldContent(field)
```
Comment (Yuliia-Dovhal) [inline src/QuickMarcEditor/utils.js:53]: Wouldn't be better to wrap single argument into brackets too? (according to the https://github.com/airbnb/javascript#arrows--paren-wrap)
```suggestion
    records: marcRecord.records.map((field) => (shouldRemoveFieldContent(field)
```

**Review by Yuliia-Dovhal — APPROVED** [review]

**Review by uladislausamets — APPROVED** [review]

**Pull request merged**